test/src/folding/parsers/highlight_parser_scala_test.dart

```diff
@@ -142,8 +144,8 @@ import users.{ User, UserPreferences}  // Only imports selected members''';
     });
 
     test('Multiline imports form braces foldable block', () {
-      //TODO(Malarg): handle scala multiline blocks
-      //https://github.com/akvelon/flutter-code-editor/issues/78.
+      // TODO(Malarg): handle scala multiline blocks
+      //  https://github.com/akvelon/flutter-code-editor/issues/78.
```

Comment:
```suggestion
      //  https://github.com/akvelon/flutter-code-editor/issues/78
```

Some IDEs may treat `.` as a part of the URL.